aws_lambda/s3_create_datasource_in_db.py

```diff
@@ -7,6 +7,7 @@
 from urllib.parse import unquote_plus
 
 from psycopg2.pool import SimpleConnectionPool
+import mimetypes
```

Comment:
mimetypes is a build-in module, so it has to be placed in the top import block (don't forget alphabet order)